resources/views/blocks/_seachResults.antlers.html

```diff
@@ -0,0 +1,50 @@
+<div class="container mx-auto py-12 px-4 max-w-4xl">
```

Comment:
- Remove "mx-auto" and "px-4" -> container already has the relevant center and horizontal padding
- Remove "max-w-4xl".. I understand why you've added this but it looks strangely squashed in the centre
- Change "py-12" to "py-8 lg:py-12 2xl:py-14" -> to match all our other blocks